commit 9d0d64df8b5569f02892b1fd8690043bea7bfde7
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Tue Jul 14 01:02:00 2020 +0800

    Fix compatibility issue in 5.3
    
    The offending entry point object looks like:
    EntryPoint(name='compute', value='openstackclient.compute.client',
    group='openstack.cli.base')
    
    Fixes https://storyboard.openstack.org/#!/story/2007917
    
    Change-Id: I0f3cc62e23efdc14203ce6645581d5ba5dbf7fa0

diff --git a/openstackclient/common/clientmanager.py b/openstackclient/common/clientmanager.py
index 66dc880e..36c3ce26 100644
--- a/openstackclient/common/clientmanager.py
+++ b/openstackclient/common/clientmanager.py
@@ -154,7 +154,10 @@ def get_plugin_modules(group):
         try:
             module_name = ep.entry_point.module_name
         except AttributeError:
-            module_name = ep.entry_point.module
+            try:
+                module_name = ep.entry_point.module
+            except AttributeError:
+                module_name = ep.entry_point.value
 
         try:
             module = importlib.import_module(module_name)
